Content.Client/Singularity/Visualizers/RadiationCollectorVisualizerComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Robust.Client.Animations;
+
+namespace Content.Client.Singularity.Visualizers;
+
+[RegisterComponent]
+[Access(typeof(RadiationCollectorVisualizerSystem))]
+public sealed class RadiationCollectorVisualizerComponent : Component
```

Comment:
visuals

Reply:
Converted to just `RadationCollectorComponent`.